src/python/WMQuality/Emulators/EmulatedUnitTestCase.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
 
         TODO: parameters to turn off emulators individually
         """
-
+        print(f"Running setUp with emulated services. ")
```

Comment:
why print and not logger call?

Reply:
No strong preference as well, but for many tests the logger buffer gets many many records, so a line like that is harder to see than if we get it in the stdout. I am fine to change it though, if you prefer.

Reply:
The strange thing that you always made similar argument when I used print in a code. I followed your suggestion here :) since if codebase use logger everywhere it seems logical to use it here too.

Reply:
I think for unit tests it gives us a good separation from the actual WMCore code, thus stdout would have only unit test related log records.